data/virt_autotest/sriov_network_guest_logging.sh

```diff
@@ -10,6 +10,10 @@ lspci
 echo ""
 ip a
 echo ""
+ls -l /etc/sysconfig/network/
+echo ""
```

Comment:
What is this ```echo ""``` ?

Reply:
for an empty line to better formatting only. :smile: